src/store/todo/todoReducer.ts

```diff
@@ -22,20 +18,52 @@ export default function todosReducer(
   action: todosTypes.TodosActionType
 ) {
   switch (action.type) {
-    case todosTypes.ADD_TODOS:
+    // --------------- ADD TODOS -----------
+    case todosTypes.ADD_TODOS_PENDING:
       return {
         ...state,
```

Comment:
This action brings nothing. Add loading state for loading indicator or something.

Reply:
Correct each action in reducer (loading, error)